content/docs/command-reference/studio/login.md

```diff
@@ -0,0 +1,45 @@
+## studio login
+
+Authenticate DVC with Studio to save the token to global [DVC configuration].
+
+[dvc configuration]:
+  /doc/user-guide/project-structure/configuration#config-file-locations
+
+## Synopsis
+
+```usage
+usage: dvc studio login [-h] [-q | -v] [-H <hostname>] [-s <scopes>] [-n <name>] [-d]
+```
+
+## Description
+
+By default, this command authenticates the DVC with Studio using default scopes
+and assigns a random name as the token name.
+
+## Options
+
+- `-H <hostname>`, `--hostname <hostname>` - the hostname of the Studio instance
+  to authenticate with. Defaults to `https://studio.iterative.ai`
+
+- `-s <scopes>`, `--scopes <scopes>` - comma separated
+  [values of scopes](#available-scopes) for the authentication token. Defaults
+  to all available scopes.
+
+- `-n <name>`, `--name <name>` - the name of the authentication token. It will
+  be used to identify token shown in Studio profile. Defaults to a random name.
+
+- `-o`, `--no-open` - use authentication flow based on user code. You will be
+  presented with user code to enter in browser. DVC will also use this if it
+  cannot launch browser on your behalf.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Available scopes
+
+- `EXPERIMENTS` - Experiment operations
+- `DATASETS` - Dataset operations
```

Comment:
What is `DATASETS`? Do we need an explanation of scopes in the Studio docs?

Reply:
Do we have any official documentation for datasets, we can link to? cc. @shcheklein

Reply:
Yes, it is. I think it is under flag, available for self hosted customer or internal users.

Reply:
We can either leave it out of docs for now or leave it in but say something like "coming soon" without going into details.

Reply:
Added the information